tests/discovery.spec.coffee

```diff
@@ -53,6 +54,18 @@ describe 'Discoverable Services:', ->
 			if dummyService.tags?
 				expect(service.tags).to.deep.equal(dummyService.tags)
 
+	inspectPublishedAddresses = (service, hostIPs) ->
```

Comment:
This function returns a boolean, so I think a different name would probably be clearer as to what actually happens, what property are we inspecting for true/false?  Generally I like the `is....` naming style for booleans as it naturally encourages a sensible name imo